pkg/interface/src/views/apps/launch/components/tiles/tile.js

```diff
@@ -1,30 +1,25 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import defaultApps from '~/logic/lib/default-apps';
-const routeList = defaultApps.map((e) => {
-  return `/~${e}`;
-});
 
 import { Box } from "@tlon/indigo-react";
 
 export default class Tile extends React.Component {
   render() {
-    const { bg, to, p, ...props } = this.props;
+    const { to, href, p, ...props } = this.props;
     
     let childElement = (
-      <Box p={typeof p === 'undefined' ? 2 : p} bg={bg} width="100%" height="100%">
+      <Box p={typeof p === 'undefined' ? 2 : p} width="100%" height="100%">
         {props.children}
       </Box>
     );
 
     if (to) {
-      if (routeList.indexOf(to) === -1) {
-        childElement= (<Link to={to}>{childElement}</Link>);
-      } else {
-        childElement= (<a href={to}>{childElement}</a>);
-      }
-      
+      childElement= (<Link to={to}>{childElement}</Link>);
+    } else if (href) {
+      childElement= (<a href={href}>{childElement}</a>);
```

Comment:
Should preserve the old logic here. It’s for third party apps which need to be a href’d.

Reply:
Or are you just passing href for those separately